lib/AST/IndexSubset.cpp

```diff
@@ -16,7 +16,6 @@ using namespace swift;
 
 IndexSubset *
 IndexSubset::getFromString(ASTContext &ctx, StringRef string) {
-  if (string.size() < 0) return nullptr;
```

Comment:
@dan-zheng This seems like a typo. Should this have been `string.empty()`?

Reply:
Yes, I think `if (string.empty()) return nullptr` was intended here. Thanks for the catch!